insert.md

```diff
@@ -26,10 +26,10 @@ table td:first-child {
 
 Parameter | Description
 ----------|------------
-`qualified_name` | The name of the table to insert into.
+`qualified_name` | The [table expression](table-expressions.html) you want to write data to.|
```

Comment:
Here unfortunately we only support a simple table name. It would be misleading to suggest we can place any table expression as target to INSERT (although other dbs do support this, we don't).